src/source/Signaling/LwsApiCalls.h

```diff
@@ -31,9 +31,12 @@ extern "C" {
 #define GET_SIGNALING_CHANNEL_ENDPOINT_API_POSTFIX "/getSignalingChannelEndpoint"
 #define DELETE_SIGNALING_CHANNEL_API_POSTFIX       "/deleteSignalingChannel"
 #define GET_ICE_CONFIG_API_POSTFIX                 "/v1/get-ice-server-config"
+#define JOIN_STORAGE_SESSION_API_POSTFIX           "/JoinStorageSession"
+#define UPDATE_MEDIA_STORAGE_CONF_API_POSTFIX      "/UpdateMediaStorageConfiguration"
 
 // Signaling protocol name
-#define SIGNALING_CHANNEL_PROTOCOL "\"WSS\", \"HTTPS\""
+#define SIGNALING_CHANNEL_PROTOCOL                 "\"WSS\", \"HTTPS\""
+#define SIGNALING_CHANNEL_PROTOCOL_W_MEDIA_STORAGE "\"WSS\", \"HTTPS\", \"WEBRTC\""
```

Comment:
This doesn't seem necessary. Instead of having that bool that is used only to check this protocol, couldn't we just change SIGNALING_CHANNEL_PROTOCOL to use this new WEBRTC string?

Reply:
If the media server is up, we can remove the old one. But right now, we need this to run samples, otherwise the legacy sample will fail.